### DISCUSSION TITLE
Intelisense to include context for variables

### DISCUSSION THREAD

**Discussion by kwnath** [stitchesjs/stitches#740] [Ideas]
I'm not sure if this is the right place to suggest this or whether it's already been done and I've just misconfigured it but it'd be nice to have a way to get the value of the variables like in tailwind

Tailwind reference: https://github.com/tailwindlabs/tailwindcss-intellisense
<img width="783" alt="Screenshot 2021-08-27 at 5 45 59 PM" src="https://user-images.githubusercontent.com/26071410/131111925-a52baf19-e547-4d03-be85-29f93c50c5df.png">

Currently it looks like this for stitches:
<img width="496" alt="Screenshot 2021-08-27 at 6 15 38 PM" src="https://user-images.githubusercontent.com/26071410/131112045-f11fc33f-7e92-408c-a15d-91b22e28fb15.png">

**Comment by peduarte** [comment]
This should work via TypeScript, doesnt it show for you?

**Reply by kwnath** [reply]
Ah, it actually doesn’t show for me. I may have misconfigured my settings
somehow.

A bit of context I’m using yarn workspaces (not sure if that matters)

**Reply by peduarte** [reply]
The screen shot you posted looks good though. It's showing the tokens suggestion list. Were you expecting to see something else?

I think the only difference is that tailwind can display the color value as a little box. But how would that work for space?

**Reply by kwnath** [reply]
Yeah I was expecting to see what tailwind displays as well. I think that it should  show the rem/px value. Because a lot of the times I just lose context of which tokens have what value...though I suppose I could resolve this issue by using tokens with meaningful names.
